extension/src/shared/view/state/colorSchemeForComponent.ts

```diff
@@ -155,7 +161,10 @@ export const makeColorSchemeAtomForComponent = (layers: readonly LayerModel[]) =
         const currentRuleId = colorScheme.useDefault
           ? colorScheme.currentRuleId ?? component.preset?.rules?.[0].id
           : colorScheme.currentRuleId;
-        const rule = component.preset?.rules?.find(rule => rule.id === currentRuleId);
+        const rule = component.preset?.rules?.find(rule => rule.id === currentRuleId) as
+          | FillColorConditionFieldPresetRule
```

Comment:
I don't want this `as` here, but `rule`'s type become `FillColorConditionFieldPresetRule` only while the `rules`'s type is `FillColorConditionFieldPresetRule[] | FillAndStrokeColorConditionFieldPresetRule[] | undefined`. I'm not sure why so i added this `as` to correct the type here.

Reply:
I fixed!
https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/155/commits/a69ff0457a52cf2b80b7259b3571d89a0f031571